int32 room_point

---
bool roompoint_check